src/bindings/js/node/include/preprocess/preprocess_wrap.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <napi.h>
+
+#include <openvino/openvino.hpp>
+
+#include "preprocess/resize_algorithm.hpp"
+#include "preprocess/pre_post_process_wrap.hpp"
+
+class PreProcessWrap {
+public:
+    /** @brief This method is called during initialization of OpenVINO native add-on.
+     * It exports JavaScript preprocess property.
+     */
+    static Napi::Object Init(Napi::Env env, Napi::Object exports);
```

Comment:
It is a bad pattern to create a class that has only static members and I think that in the future we will only add more static methods if any. 
In OOP a class is a construct that is used as a blueprint to create instances of itself. Here we won't do it.